### PR TITLE
Course Offline functionality

### PR DESCRIPTION
**Pull request by ikorgik**

### CODE REVIEW

**Comment by cneuro** [comment]
@ikorgik is this up to date with PR #20 ? I can only see one of the commits

**Comment by ikorgik** [comment]
> @ikorgik is this up to date with PR #20 ? I can only see one of the commits

Yes, it was based on that PR

File: js/src/components/05_pages/Course/index.js [hunk 1]
```diff
@@ -59,6 +63,14 @@ const StyledShadowBox = withStyles(theme => ({
   }
 }))(Box);
 
+const StyledBox = withStyles(theme => ({
+  root: {
+    marginTop: theme.spacing(2),
+    alignItems: 'center',
+    justifyContent: 'space-around',
```
Comment (cneuro) [inline js/src/components/05_pages/Course/index.js:70]: change to `center`

Comment (cneuro) [inline js/src/components/05_pages/Course/index.js:70]: To better center (1): Replace `space-around` with `center`.

File: js/src/components/05_pages/Course/index.js [hunk 2]
```diff
@@ -125,6 +139,14 @@ const Course = ({ node, width }) => {
               <Image src={node.image.url} alt={node.title}/>
             </Hidden>
             }
+
+            {pwaSettings &&
+            <StyledBox display="flex">
+              <Typography variant="body1">For learning offline</Typography>
```
Comment (cneuro) [inline js/src/components/05_pages/Course/index.js:145]: To better center (2): Replace with new `StyledTypography`:

```
const StyledTypography = withStyles(theme => ({
  root: {
    marginRight: theme.spacing(2),
  }
}))(Typography);
```

**Pull request merged**